allennlp/data/dataset_readers/semantic_role_labeling.py

```diff
@@ -46,11 +99,35 @@ class SrlReader(DatasetReader):
     def __init__(self,
                  token_indexers: Dict[str, TokenIndexer] = None,
                  domain_identifier: str = None,
-                 lazy: bool = False) -> None:
+                 lazy: bool = False,
+                 bert_model_name: str = None) -> None:
         super().__init__(lazy)
         self._token_indexers = token_indexers or {"tokens": SingleIdTokenIndexer()}
         self._domain_identifier = domain_identifier
 
+        if bert_model_name is not None:
+            self.bert_tokenizer = BertTokenizer.from_pretrained(bert_model_name)
+            self.lowercase_input = "uncased" in bert_model_name
+        else:
+            self.bert_tokenizer = None
+            self.lowercase_input = False
+
+    def _tokenize_input(self, tokens: List[str]) -> Tuple[List[str], List[int]]:
```

Comment:
again, add a docstring here indicating what this does and that it's only used in the bert case.

also, I'm not convinced this is the best name, maybe `_convert_tokens_to_wordpieces` ?